2-mandatory/6-journey-planner.js

```diff
@@ -92,7 +97,9 @@ function isAccessibleByTransportMode() {}
    - Returns the name of the location
       e.g: "Tower Bridge"
 */
-function getLocationName() {}
+function getLocationName(arr) {
+  return arr[0];
```

Comment:
This is already a good answer in here. 

However, I just like to point out that, in here we assume `[0]` is always exists in `arr`. If `arr` is empty, `undefined` will be returned. There is actually a better (safer) way to get the first element in array: https://stackoverflow.com/a/27180125/12219600

Reply:
yourArray=[undefined,1,2,3,4]
let firstItem = yourArray.find(x=>x!==undefined);
firstItem

these correct?

Reply:
It would be even safer if you can filter out the `null` values:

```
const yourArray=[undefined,null,1,2,3,4]
const finteredArray = yourArray.filter(x=>x != null);
console.log(finteredArray) // both undefined and null values are filtered out
```